extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -40,4 +43,129 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
 amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
   border-radius: 16px 16px 0px 0px !important;
   font-family: Poppins, sans-serif !important;
+  font-weight: 400 !important;
+  background: white !important;
+  color: black !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-banner {
+  position: relative !important;
+  width: 100% !important;
+  border-radius: 16px 16px 0px 0px !important;
+  font-size: 24px !important;
+  text-align: center !important;
+  height: 44px !important;
+  background: #EDEDED !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-content {
+  position: relative !important;
+  margin: 28px 28px 24px 28px !important;
+}
+
+.i-amphtml-story-subscriptions-publisher-text {
+  position: relative !important;
+  margin-bottom: 41px !important;
+}
```

Comment:
We could remove this if we remove the wrapper. 
The margin bottom could be on the last text element.
position relative might not be needed here.